nixpkgs_review/nix.py

```diff
@@ -204,3 +377,38 @@ def write_shell_expression(filename: Path, attrs: List[str], system: str) -> Non
 }
 """
         )
+
+
+def strip_ansi_colors(s: str) -> str:
+    # https://stackoverflow.com/a/14693789/1079728
+    # 7-bit C1 ANSI sequences
+    ansi_escape = re.compile(
+        r"""
+        \x1B  # ESC
+        (?:   # 7-bit C1 Fe (except CSI)
+            [@-Z\\-_]
+        |     # or [ for CSI, followed by a control sequence
+            \[
+            [0-?]*  # Parameter bytes
+            [ -/]*  # Intermediate bytes
+            [@-~]   # Final byte
+        )
+    """,
+        re.VERBOSE,
+    )
+    return ansi_escape.sub("", s)
+
+
+def _store_dir() -> str:
```

Comment:
Can well tell python to cache this? The nix store path will never change while running.

Reply:
It's only called once anyways, but yes I can add a `@functools.lru_cache`.